data/json/npcs/godco/cook.json

```
@@ -0,0 +1,267 @@
+    "id": "TALK_GODCO_cook_ask_membership",
+    "dynamic_line": {
+      "days_since_cataclysm": 20,
+      "yes": "You have to ask our leader, Helena, first.  She's the one who makes those decisions.  But as I said, your chances are low, like everyone else's.  The newest member joined just a long time ago.",
+      "no": "You have to ask our leader, Helena, first.  She's the one who makes those decisions.  But as I said, your chances are low, like everyone else's. If you had found our community earlier, you could have had a higher chance to join.  The newest member joined just a few days ago."
+    },
+    "responses": [ { "text": "Atleast I tried.", "topic": "TALK_GODCO_cook_1" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_cook_ask_goods",
+    "dynamic_line": "From me?  Nothing.  But we have a tailor, herbalist and a hunter.  Go see their products yourself, maybe you find something you need.  Keep in mind that we don't use regular dollars here.",
+    "responses": [
+      { "text": "I'll take a look.", "topic": "TALK_GODCO_cook_1" },
+      { "text": "What do you use as money?", "topic": "TALK_GODCO_cook_currency" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_cook_ask_past",
+    "dynamic_line": "You know, I used to work as a cook in a homeless shelter.  Coincidentally, that's how I met the community.  They used to go there many times to help the residents and preach from the Bible.  I am sure this is why my family and I could join.  Also, working in the shelter taught me to cook on a very low budget.  It's an even more useful skill today.",
+    "responses": [ { "text": "Let's talk about something else.", "topic": "TALK_GODCO_cook_1" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_cook_ask_mood",
+    "dynamic_line": "Could be better, could be worse.  But I'm not complaining.  I have everything I need here: my family, a nice job and roof over my head.  I wish God was as forgiving to others as to me.",
+    "responses": [ { "text": "It's good to hear that.", "topic": "TALK_GODCO_cook_1" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_cook_currency",
+    "dynamic_line": "We use religious icons as our currency.  These were made before <the_cataclysm>, and the name of our community is written on the back.  It's a lot more handy than old-world dollars, those are basicly just firestarters here.",
+    "responses": [
+      { "text": "Oh, okay.", "topic": "TALK_GODCO_cook_1" },
+      { "text": "What can I buy here?", "topic": "TALK_GODCO_cook_ask_goods" }
+    ]
+  },
+  {
+    "id": "MISSION_GODCO_COOK_FOOD_1",
+    "type": "mission_definition",
+    "name": "Picky Meals",
+    "description": "Find 100 salt.",
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 1,
+    "value": 0,
+    "item": "salt",
+    "count": 100,
+    "origins": [ "ORIGIN_SECONDARY" ],
+    "followup": "MISSION_GODCO_COOK_FOOD_2",
+    "dialogue": {
+      "describe": "I need 100 salt.",
```

I guess precise number isn't good for job offer. But that's IMO.
```suggestion
      "describe": "I need a supply of salt.",
```